src/main/java/core/basesyntax/FigureColor.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class FigureColor implements Figure {
+    private String color;
+
+    public FigureColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    public void draw() {
+    }
```

Comment:
The 'getArea' and 'draw' methods should not be implemented here if this class is only responsible for color. These methods should be implemented in the concrete figure classes that extend a base figure class or implement the 'Figure' interface directly.